Exercise_5.py

```diff
@@ -0,0 +1,48 @@
+class IterativeQuickSort:
+	def swap(self, arr, i, j):
+		pass
+
+	def partition(self, arr, low, high):
+		pivot = arr[high]
+		i = low - 1
+		for j in range(low, high):
+			if (arr[j] <= pivot):
+				i += 1
+				arr[i], arr[j] = arr[j], arr[i]
```

Comment:
you could have also used the swap function here
by writing the code in swap function